Client/Extensions/UIImageViewExtensions.swift

```diff
@@ -16,6 +16,13 @@ extension UIColor {
         getRed(&r, green: &g, blue: &b, alpha: &a)
         return (r, g, b, a)
     }
+
+    func image(_ size: CGSize = CGSize(width: 1, height: 1)) -> UIImage {
```

Comment:
Added simple method to make any coloured images. This is for default white image for blank tab or when we don't have any screenshot.